swagger.json

```diff
@@ -0,0 +1,1344 @@
+          "x-go-name": "Href"
+        },
+        "id": {
+          "type": "string",
+          "x-go-name": "ID"
+        },
+        "last_failure_message": {
+          "type": "string",
+          "x-go-name": "LastFailureMessage"
+        },
+        "last_run_timestamp": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "LastRunTimestamp"
+        },
+        "max-failures": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "MaxFailures"
+        },
+        "miss_count": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "MissCount"
+        },
+        "name": {
+          "type": "string",
+          "x-go-name": "Name"
+        },
+        "schedule": {
+          "$ref": "#/definitions/Schedule"
+        },
+        "start": {
+          "type": "boolean",
+          "x-go-name": "Start"
+        },
+        "task_state": {
+          "type": "string",
+          "x-go-name": "TaskState"
+        },
+        "version": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "Version"
+        },
+        "workflow": {
+          "$ref": "#/definitions/WorkflowMap"
+        }
+      },
+      "x-go-package": "github.com/intelsdi-x/snap/mgmt/rest/v2"
+    },
+    "Tasks": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/Task"
+      },
+      "x-go-package": "github.com/intelsdi-x/snap/mgmt/rest/v2"
+    },
+    "WorkflowMap": {
+      "description": "WorkflowMap represents a map of a desired workflow that is used to create a scheduleWorkflow",
+      "type": "object",
+      "required": [
+        "collect"
+      ],
+      "properties": {
+        "collect": {
+          "$ref": "#/definitions/CollectWorkflowMapNode"
+        }
+      },
+      "x-go-package": "github.com/intelsdi-x/snap/scheduler/wmap"
+    },
+    "metricInfo": {
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "Version_"
+        }
+      },
+      "x-go-package": "github.com/intelsdi-x/snap/scheduler/wmap"
+    }
+  },
+  "responses": {
+    "ErrorResponse": {
+      "description": "ErrorResponse represents the Snap error response type.\n\nIt includes an error message and a map of fields.",
+      "schema": {
+        "$ref": "#/definitions/Error"
+      }
+    },
+    "MetricsResponse": {
+      "description": "MetricsResp is the representation of metric operation response.",
+      "schema": {
+        "type": "object",
+        "properties": {
+          "metrics": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/Metric"
+            },
+            "x-go-name": "Metrics"
+          }
+        }
+      }
+    },
+    "PluginConfigResponse": {
+      "description": "PluginConfigItem represents the response of a plugin config items.",
```

Comment:
@candysmurf, please verify if the description is as you expected. I have a doubt about PluginConfigItem - maybe it should be PluginConfigResponse.